### PR TITLE
Josh/cs

### PR DESCRIPTION
**Pull request by Joshua-Ogbonna**

### CODE REVIEW

File: components/ui/organisms/ComingSoon.jsx [hunk 1]
```diff
@@ -185,6 +189,54 @@ export default function ComingSoon() {
...
+          </div>
+
+          {/* Team Members */}
+          <div className="flex align-middle justify-between gap-5 flex-wrap md:justify-center sm:justify-center">
+            {team.map((member) => (
+              <div key={member.name} className="bg-[#C3FFD0] w-[380px] shadow-[4px 8px 24px rgba(170, 170, 170, 0.6)] cursor-pointer py-4 px-2 rounded-2xl mb-4">
+                <Image src={member.image} alt={member.name} className="w-full" />
+                <div className="flex align-top justify-between mt-4"> 
+                  <div>
+                    <div className="text-[#00391B] font-semibold text-3xl">{member.name}</div>
```
Comment (CeoFred) [inline components/ui/organisms/ComingSoon.jsx:230]: The text size should be different on mobile.. text-xl is just fine, or just play around and see which one fits fine.

File: components/ui/organisms/ComingSoon.jsx [hunk 2]
```diff
@@ -185,6 +189,54 @@ export default function ComingSoon() {
...
+                    <div className="text-lg text-[#006D3A] font-normal">{member.role}</div>
+                  </div>
+                  <div className="flex gap-2">
+                    <Image className="w-[32px] h-[32px]" src={Twitter} alt="twitter" />
```
Comment (CeoFred) [inline components/ui/organisms/ComingSoon.jsx:234]: this height and width for icons should be different for smaller screens

**Review by CeoFred — CHANGES_REQUESTED** [review]

**Comment by CeoFred** [comment]
@dpkreativ for the carousel, what library are you gonna suggest?

File: components/ui/organisms/ComingSoon.jsx (hunk 1 above)
Comment (Joshua-Ogbonna) [reply components/ui/organisms/ComingSoon.jsx]: Done

File: components/ui/organisms/ComingSoon.jsx (hunk 2 above)
Comment (Joshua-Ogbonna) [reply components/ui/organisms/ComingSoon.jsx]: This also is done

**Comment by dpkreativ** [comment]
> @dpkreativ for the carousel, what library are you gonna suggest?

I suggested Swyper js in the Trello card

**Comment by dpkreativ** [comment]
[Create Coming Soon page](https://trello.com/c/UiMCo6L7/3-create-coming-soon-page)

**Comment by dpkreativ** [comment]
Hi @Joshua-Ogbonna, you haven't been responsive for a while. I will be closing this PR in the next 24 hours if there's no response to this message.

**Comment by Joshua-Ogbonna** [comment]
I'm sorry for the silence @dpkreativ. I've been down kinda. Would jump and complete before Wednesday

**Comment by dpkreativ** [comment]
Hey man @Joshua-Ogbonna, sorry about your health. It's okay to take as much time off as you need.
Thanks for your contributions so far, I hope you get well quickly.

@CeoFred kindly merge this PR so we can keep working from where @Joshua-Ogbonna stopped

**Comment by dpkreativ** [comment]
@CeoFred still waiting for you to merge this PR

**Pull request merged**